plugins/3MFReader/WorkspaceDialog.qml

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) 2022 Ultimaker B.V.
 // Cura is released under the terms of the LGPLv3 or higher.
 
-import QtQuick 2.10
+import QtQuick 2.14
```

Comment:
Why update this? You aren't using antything from a newer QtQuick version. This import only defines what functions are available. Increasing it doesn't mean you suddenly get bugfixes you wouldn't otherwise.

Reply:
Right, I needed it at some point when playing with the table view/model items but didn't change it back.